inception/inception-telemetry/src/main/java/de/tudarmstadt/ukp/clarin/webanno/telemetry/matomo/MatomoTelemetryTraitsEditor_fr.html

```diff
@@ -0,0 +1,66 @@
+<!--
```

Comment:
I wonder if it would be better to pull out the strings here and put them into a `.properties` file instead of having multiple versions of the `.html` files around.

Reply:
Thanks for trying this out. Turns out it was a bad idea because the extracted strings contain HTML markup which would force use disabling HTML escaping on the resource models - which we do not want to do. So better to translate the entire HTML files as you had. I have added a corresponding todo to the initial description above.